config/examples/sandbox_proxy.yml

```diff
@@ -14,8 +15,9 @@ base: &default
   override: 'http://localhost:8091'
   sandbox_endpoint: 'https://%{system_name}-%{account_id}.staging.%{env}apicast.io:%{port}'
   hosted_proxy_endpoint: 'https://%{system_name}-%{account_id}.%{env}apicast.io:%{port}'
-  apicast_staging_endpoint: 'http://%{system_name}-%{account_id}.staging.%{env}apicast.dev:8080'
-  apicast_production_endpoint: 'http://%{system_name}-%{account_id}.%{env}apicast.dev:8080'
+  # Apicast endpoints as visible from the client
+  apicast_staging_endpoint: 'http://%{system_name}-%{account_id}.staging.%{env}apicast.dev.localhost:8080'
+  apicast_production_endpoint: 'http://%{system_name}-%{account_id}.%{env}apicast.dev:8080.localhost'
```

Comment:
This one is not right :upside_down_face: 

I guess you meant `apicast.dev.localhost:8080` as above?